packages/react-tabster/src/hooks/useFocusIndicatorStyle.ts

```diff
@@ -12,5 +12,14 @@ export const createFocusIndicatorStyleRule = (
   ':focus-visible': {
     outline: 'none',
   },
-  [KEYBOARD_NAV_SELECTOR]: typeof rule === 'function' ? rule(theme) : rule,
+  [`${KEYBOARD_NAV_SELECTOR} :focus`]: typeof rule === 'function' ? rule(theme) : rule,
+});
+
+export const createFocusWithinIndicatorStyleRule = (
```

Comment:
- cant we make original function configurable or curried to ship less bytes? this seems like unnecessary duplication.
- can we add tests?

Reply:
> * cant we make original function configurable or curried to ship less bytes? this seems like unnecessary duplication.

What would you suggest?! currying or options for configuration?1 for me both are good options

> * can we add tests?

That's actually harder than it looks, would require some e2e test perhaps

Reply:
> What would you suggest?! currying or options for configuration?1 for me both are good options

hmm good question :D

- I'd be consistent in terms of our apis (haven't seen currying in vNext )
- creating closures via currying can be more DRY/although might have bigger memory perf impact


so config argument ? WDYT (I'd make it as 1st arg param though  - easy to migrate to curried API later... )

Reply:
Making an optional argument as the first one meanwhile there's already required argument is kind of hard hehehe. I'll keep it as a second argument for now